index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Document</title>
```

Comment:
[Optional] Consider adding a descriptive title to your HTML file to increase SEO performance.

[A bit more on title tags and SEO](https://moz.com/learn/seo/title-tag)